SECURITY.md

```diff
@@ -73,13 +73,13 @@ severity.
 If a vulnerability does not affect any point release but only master, additional caveats apply:
 
 * If the issue is detected and a fix is available within 5 days of the introduction of the
-  vulnerability, the fix will be publicly reviewed and landed on master. A courtesy e-mail will be
-  sent to envoy-users@googlegroups.com, envoy-dev@googlegroups.com,
-  envoy-security-announce@googlegroups.com and cncf-envoy-distributors-announce@lists.cncf.io if 
-  the severity is medium or greater.
-* If the vulnerability has been in existence for more than 5 days, we will activate the security
-  release process for any medium or higher vulnerabilities. Low severity vulnerabilities will still
-  be merged onto master as soon as a fix is available.
+  vulnerability, or the issue is deemed a low severity vulnerability, the fix will be publicly
+  reviewed and landed on master. A courtesy e-mail will be sent to envoy-users@googlegroups.com,
+  envoy-dev@googlegroups.com, envoy-security-announce@googlegroups.com and
+  cncf-envoy-distributors-announce@lists.cncf.io if the severity is medium or
```

Comment:
If the severity is medium or greater at maintainer discretion, a courtesy e-mail will be sent to ...